pyronear/models/ssresnet.py

```diff
@@ -1,64 +1,49 @@
-from torchvision.models.resnet import ResNet, BasicBlock, Bottleneck
+from torchvision.models.resnet import ResNet, resnet18, BasicBlock
 import torchvision
 from torch import nn
 import torch
 import torch.nn.functional as F
 
 
+__all__ = ["SSResNet"]
```

Comment:
ssresnet18 missing here, isn't it? :)

Reply:
No, I removed it on purpose. Otherwise this test fail : https://github.com/pyronear/PyroNear/blob/ec3ba3116135d853e5eb10836ba509c9e04462bc/test/test_models.py#L62

I will put it back when a pretrained model will be available

Reply:
Good catch! But then, you might have to remove it from the documentation?

Reply:
Yes maybe it's better for know